### PR TITLE
Enforcing memory and disk space restrictions from seed spec (#41)

### PR DESCRIPTION
**Pull request by JohnPTobe**

### CODE REVIEW

File: cli/main.go [hunk 1]
```diff
@@ -1362,7 +1413,7 @@ func CheckSudo() {
 
 //DockerVersionHasLabel returns if the docker version is greater than 1.11.1
 func DockerVersionHasLabel() bool {
-	cmd := exec.Command("docker", "--version")
+	cmd := exec.Command("docker", "version", "-f", "{{.Client.Version}}")
```
Comment (gisjedi) [inline cli/main.go:1421]: Good change. 💯

File: cli/main.go [hunk 2]
```diff
@@ -916,6 +926,11 @@ func DefineInputs(seed *objects.Seed) ([]string, error) {
 
 		// Expand input VALUE
 		val = GetFullPath(val)
+		info, err := os.Stat(val)
+		if os.IsNotExist(err) {
+			fmt.Fprintf(os.Stderr, "ERROR: Input file %s not found\n", val)
+		}
+		size += (1.0 * float64(info.Size())) / (1024.0 * 1024.0)
```
Comment (gisjedi) [inline cli/main.go]: This is fairly self evident, but I think a comment would be justified here or in func comment to indicate sizes will be in MiBs

File: cli/main.go [hunk 3]
```diff
@@ -1098,13 +1113,49 @@ func DefineSettings(seed *objects.Seed) ([]string, error) {
...
+	var disk float64
+	
+	for _, s := range seed.Job.Interface.Resources.Scalar {
+		if s.Name == "mem" {
+			mem := s.Value + s.InputMultiplier * size
+			intMem := int64(math.Ceil(mem))
+			resources = append(resources, "-m")
+			resources = append(resources, fmt.Sprintf("%dm", intMem))
+		}
+		if s.Name == "disk" {
```
Comment (gisjedi) [inline cli/main.go]: Again I think a comment here or in func header (probably in header to accomdate both mem/disk) to indicate what the goal of the computation is would be justified. Basically, "Compute the total output disk / memory resource requirements as a function of input file size."

**Review by gisjedi — CHANGES_REQUESTED** [review]
A few small comments. Great work!

**Comment by JohnPTobe** [comment]
Good suggestions. Done. 👍

**Review by gisjedi — APPROVED** [review]
LGTM

**Pull request merged**